pkg/virt-launcher/virtwrap/efi/efi.go

```diff
@@ -42,33 +44,50 @@ type EFIEnvironment struct {
 	varsSecureBoot string
 	codeSEV        string
 	varsSEV        string
+	codeTDX        string
+	codeTDXSB      string
 }
 
-func (e *EFIEnvironment) Bootable(secureBoot, sev bool) bool {
+func (e *EFIEnvironment) Bootable(secureBoot, sev bool, tdx bool) bool {
```

Comment:
the SNP code touches this too and we will soon have

```
func (e *EFIEnvironment) Bootable(secureBoot, sev bool, snp bool, tdx bool) bool {
```

it does not scale.  Perhaps we should move this param to an enum-like type.

Reply:
what about 

`func (e *EFIEnvironment) Bootable(secureBoot, sectype string) bool {`

and we may introduce some local functions like
`
is_sev(sectype string)
is_snp(sectype string)
is_tdx(sectype string)
`
or

use a switch .. case..

cc @alancaldelas